content/tul/funksjonalitet/rolleadministrasjon.md

```diff
@@ -74,13 +74,24 @@ Eksterne roller tildeles automatisk til sluttbrukere basert på informasjon i Fo
 eksterne roller, slik at innehaver av en bestemt ekstern rolle også får de(n) Altinn-rolle(r) som er koblet mot denne. I TUL kobles en
 Altinn-rolle til eksterne roller ved å velge en eller flere eksterne roller i en nedtrekksliste og klikke på *Legg til*-knappen.
 
+Altinn-rollene kan som regel delegeres videre; dette kan settes som en parameter ved opprettelse av rollen. Altinn-roller er ment som en
+enkel mekanisme for å delegere flere rettigheter samlet, gruppert i henhold til arbeidsdelingen i representative virksomheter.
+
+Det finnes noen unntak på roller som ikke tildeles eksterne roller fra Enhetsregisteret og Folkeregisteret. Disse rollene er innført for å ivareta behov for tilgangsstyring til ressurser som personer i roller fra Enhetsregisteret ikke nødvendigvis skal ha tilgang til (f.eks taushetsbelagte meldinger, API-ressurser o.l.)
+Rollene dette gjelder;
+- Barneroller under "Hovedrolle for sensitive tjenester"
```

Comment:
Jeg ville endret dette til: 
- "Roller for sensitive tjenester"
   - Tjenestespesifikke roller som gir tilgang til taushetsbelagte tjenester og som er lagt under "Hovedrolle for sensitive tjenester"